src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-snapshots-list/cephfs-subvolume-snapshots-form/cephfs-subvolume-snapshots-form.component.html

```diff
@@ -0,0 +1,99 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container i18n="form title"
+                class="modal-title">{{ action | titlecase }} {{ resource | upperFirst }}</ng-container>
+
+  <ng-container class="modal-content"
+                *cdFormLoading="loading">
+    <form name="snapshotForm"
+          #formDir="ngForm"
+          [formGroup]="snapshotForm"
+          novalidate>
+      <div class="modal-body">
+        <div class="form-group row">
+          <label class="cd-col-form-label required"
+                 for="snapshotName"
+                 i18n>Name</label>
+          <div class="cd-col-form-input">
+            <input class="form-control"
+                   type="text"
+                   placeholder="Snapshot name..."
+                   id="snapshotName"
+                   name="snapshotName"
+                   formControlName="snapshotName"
+                   autofocus>
+            <span class="invalid-feedback"
+                  *ngIf="snapshotForm.showError('snapshotName', formDir, 'required')"
+                  i18n>This field is required.</span>
+            <span class="invalid-feedback"
+                  *ngIf="snapshotForm.showError('snapshotName', formDir, 'notUnique')"
+                  i18n>The subvolume already exists.</span>
+            <span *ngIf="snapshotForm.showError('snapshotName', formDir, 'pattern')"
+                  class="invalid-feedback"
+                  i18n>Subvolume name can only contain letters, numbers, '.', '-' or '_'</span>
+          </div>
+        </div>
+
+        <!-- Volume name -->
+        <div class="form-group row">
+          <label class="cd-col-form-label"
+                 for="volumeName"
+                 i18n>Volume name</label>
+          <div class="cd-col-form-input">
+            <input class="form-control"
+                   id="volumeName"
+                   name="volumeName"
+                   formControlName="volumeName">
+          </div>
+        </div>
+
+        <!--Subvolume Group name -->
+        <div class="form-group row">
+          <label class="cd-col-form-label"
+                 for="subvolumeGroupName"
+                 i18n>Subvolume group
+          </label>
+          <div class="cd-col-form-input">
+            <select class="form-select"
+                    id="subvolumeGroupName"
+                    name="subvolumeGroupName"
+                    formControlName="subvolumeGroupName"
+                    #selection
+                    (change)="onSelectionChange(selection.value)"
+                    *ngIf="subVolumeGroups">
+              <ng-container *ngFor="let subvolumegroup of subVolumeGroups">
+                <option *ngIf="subvolumegroup == ''"
+                        value="">_nogroup</option>
```

Comment:
```suggestion
                <option *ngIf="subvolumegroup == ''"
                        value="">_nogroup</option>
```

I think we should start naming it differently, other than `_nogroup`, I would rather go with `Default` or something on the lines.

Reply:
the reason we moved away from it is because the user can create his own subvolume group called `Default` and we can't do anything about it. So it won't look good to make adjustments just in the UI for these namings.

Reply:
Oh, I see.. thanks for letting me know